### ISSUE TITLE
Dailymotion Support

### ISSUE THREAD

**Issue by swiretvweb** [sampotts/plyr#682]

**Comment by abazeed** [comment]
You have the best open source player yet. Adding support to **Dailymotion** will push you ahead away from other players, please put more efforts on this.

**Comment by kevinfaguiar** [comment]
+1 for Dailymotion support.

**Comment by adeeb1** [comment]
Any updates here? We'd love Dailymotion support.

**Comment by akuma06** [comment]
I'm planning to add dailymotion support for my own use, should I make a PR @friday ?

**Comment by friday** [comment]
@akuma06 It's up to @sampotts. There were architectural reasons we wanted that to be moved to plug in support. Maintaining providers is hugely problematic. I don't think that has changed, but I'm no longer contributing to plyr's development.

**Comment by akuma06** [comment]
Thanks for the reply! So I have pushed a new branch to my fork of the develop branch implementing DailyMotion (https://github.com/akuma06/plyr/tree/dailymotion). I have tested it locally and I will use it for my own app. It comes down now to @sampotts wether he wants me to PR it now or wants to wait for his plugin system. :)

**Comment by adeeb1** [comment]
Thanks @akuma06, I would love for your work to be in PR. We are starting to integrate Plyr in our application, and we would want immediate Dailymotion support. +1 from me

**Comment by abazeed** [comment]
Hello @sampotts, are you planning to pull @akuma06 's changes? We are patiently waiting to have this feature live and hope to push this a step forward at the soonest—many thanks.

**Comment by adeeb1** [comment]
I wanted to chime in here again to thank @akuma06 for the wonderful work! The Dailymotion support works perfectly, and I'm happily using it in my app through my own fork of this repo. It would be wonderful to have it merged into the official repo.

**Comment by abazeed** [comment]
Yes @adeeb1, It would be wonderful to have it merged into the official repo 🥺

**Comment by abazeed** [comment]
@sampotts congrats on the new release. wasn't there any chance to consider this in this new release? or any upcoming ones, please?

**Comment by sampotts** [comment]
Happy to take a look at a PR for it...

**Comment by abazeed** [comment]
> Happy to take a look at a PR for it...

@akuma06, could you please PR your changes?

**Comment by akuma06** [comment]
I don't mind doing the pr for it after I've checked if there are any changes with the new release. I didn't do it before since I thought @sampotts wanted to prioritize a plugin system before adding any new platform to the code base.
Will submit a pr this week :).

**Comment by abazeed** [comment]
Any updates @akuma06?
I know you may be busy with other work, and I hope you find the time for this.

**Comment by akuma06** [comment]
Sorry I completely forgot about this :'). I just updated the branch to last changes, I've checked and everything works, I've just sent a PR #2136 !

**Comment by abazeed** [comment]
Many thanks @akuma06 ❤️

**Comment by ssuunaian** [comment]
Hello @akuma06, @abazeed and others, can you please help me on how to use it on the HTML side?

**Comment by adeeb1** [comment]
> Hello @akuma06, @abazeed and others, can you please help me on how to use it on the HTML side?

You would use it just like you use the standard library. I'd need more details to provide specific help.

**Comment by onigetoc** [comment]
> > Hello @akuma06, @abazeed and others, can you please help me on how to use it on the HTML side?
> 
> You would use it just like you use the standard library. I'd need more details to provide specific help.

Sometime it's discouraging that no one or the author do not a minimal help and just fork the readme without minimal change with no help at all.

**Comment by sampotts** [comment]
Not sure what you're getting at but sorry, I don't have much time to devote to Plyr. I now have two kids (6 year old and 2 month old) and a full time job (SDE3 at Amazon) so I don't have a lot of spare time. I'm hoping to put some time into Vidstack which will hopefully be the logical successor to Plyr.

**Comment by sampotts** [comment]
Also, the current structure of Plyr means everyone gets all the logic, even for features they don't need. So, if we add every feature that anyone asks for, the player becomes bloated and folks have something new to complain about!

**Comment by onigetoc** [comment]
> Not sure what you're getting at but sorry, I don't have much time to devote to Plyr. I now have two kids (6 year old and 2 month old) and a full time job (SDE3 at Amazon) so I don't have a lot of spare time. I'm hoping to put some time into Vidstack which will hopefully be the logical successor to Plyr.

I can understand that. 
But with a popular repo like this, you may eventually have to add some good contributors who have more time to spend on it?
Like, the guy who made the Dailymotion addon for example, and i'm sure you spotted some good one until now".

Or giving it to the community with chosen top contributor?

For my complain, it was about the one you did the Dailymotion "addon" without minimal infos on is fork. 
Let's go, may be 30 more minutes or less of work in a lifetime.

I think a plugins system like Videojs would be fun but i know, more work to come :)

I checked Vime sometime days/month ago and it have good potential but there's still a lot of work to do i think to be ready et mature like Plyr.

Videojs had used a lot of free contributors and took the best Videojs plugins and functions to update Videojs and their own compagny Brightcove.

(sorry fo my english)

Bye
